test/Tests/Adapters/Engine/AbstractTypeConstructionTests.cs

```diff
@@ -31,7 +31,7 @@ public async Task Can_Execute_Query_On_Schema_From_Abstract_Types()
                 .Execute();
 
             result.ShouldHaveNoErrors();
-            result.Data.ShouldHaveFieldWithValue("commonField", new DateTime(1970, 1, 1));
+            result.Data.ShouldHaveFieldWithValue("commonField", "1970-01-01");
```

Comment:
FYI, this tracks down to the following change in `graphql-dotnet`: https://github.com/graphql-dotnet/graphql-dotnet/pull/665/files#diff-6ecb3c17fef816d6cd02a67ad408ea9eR19